src/components/stats/jacob/crop-stats.svelte

```diff
@@ -34,13 +35,31 @@
 			.map(([key]) => key)
 	);
 
-	const contests = $derived(
-		crops.length === 0 ? Object.values(contestsByCrop).flat() : crops.flatMap((c) => contestsByCrop[c] ?? [])
-	);
+	let sortBy = $state<string>('recent');
 
-	const recentContests = $derived(
-		contests?.sort((a, b) => (b?.timestamp ?? 0) - (a?.timestamp ?? 0)).slice(0, 30) ?? []
-	);
+	const contests = $derived(() => {
```

Comment:
You should probably use `$derived.by` instead, which is the built in way to do this